client/lib/site/utils.js

```diff
@@ -175,9 +175,22 @@ export default {
 	},
 
 	/**
+	 * Returns the supplied URL without the initial http(s).
+	 * @param  {String}  url The URL to remove http(s) from
+	 * @return {?String}     URL without the initial http(s)
+	 */
+	withoutHttp( url ) {
```

Comment:
This fn is not used in the PR but I think it's a good addition to Calypso as I've seen `.replace( /^https?:\/\//, '' )` on too many places.